lib/if.c

```diff
@@ -1286,16 +1286,28 @@ DEFPY_YANG (no_interface,
 	return nb_cli_apply_changes(vty, "%s", xpath_list);
 }
 
-static void netns_ifname_split(const char *xpath, char *ifname, char *vrfname)
+/*
+ * sometimes the interface name passed is just the interface name `ifname`
+ * sometimes it's `<vrf name>:<interface>`
```

Comment:
I don't think this is true. At least it shouldn't be. For vrf-lite it should always be just ifname, and for netns it should always be netns:ifname.